styles.css

```diff
@@ -8,61 +8,65 @@
   display: flex;
   flex-direction: column;
   align-items: center;
+  width: 375px;
 }
 
 .toolbar-container {
   display: flex;
   flex-direction: row;
   justify-content: center;
-  width: 90%;
+  width: 70%;
 }
 
 .logo {
   margin-right: 60%;
   background-repeat: no-repeat;
-  width: 200px;
-  height: 74px;
-  height: 28px;
-  font-family: 'Inter',sans-serif;
```

Comment:
- [ ] As requested by the previous reviewers, kindly make your design match the figma template. You can use the 'dev tools' in your browser to see what your mobile version looks like.

- [ ] Kindly Fix the link to the background image so that it displays correctly.
### Your design
![1](https://user-images.githubusercontent.com/94551484/192034555-7c22c15d-9555-4441-8323-1beb8d9e29c0.png)
### Figma template
<img width="377" alt="f1" src="https://user-images.githubusercontent.com/94551484/192034602-0b6c5837-0ac7-44b9-b760-c959cce5a775.png">